content/terraform-plugin-framework/v1.15.x/docs/plugin/framework/resources/identity.mdx

```diff
@@ -0,0 +1,194 @@
+---
+page_title: Identity
+description: >-
+Learn how to implement resource identity in the Terraform plugin framework.
+---
+
+# Resource identity
+
+A resource identity is a data object, determined by the provider, that is stored alongside the resource state to uniquely identify a remote object. A resource identity should have the following properties:
+- The resource identity must correspond to at most one remote object per provider, across all instances of that provider.
+- Given a resource identity (during `import`), the provider must be able to determine whether the corresponding remote object exists, and if so, return the resource state.
+- Unless the resource identity schema is being upgraded, the identity data must not change during the lifecycle of the remote object.
+- That is, from the creation of the remote object in the remote system until its destruction.
+
+## Resource Identity Schema
+
+To define the identity object for a managed resource, a resource identity [schema](/terraform/plugin/framework/handling-data/schemas) is provided that consists of a map of attribute names and associated behaviors. This is similar to the existing resource state schema, with a few key differences:
+
+1. The identity schema only supports primitive types and list types such as:
+    - `BoolAttribute`
+    - `Float32Attribute`
+    - `Float64Attribute`
+    - `Int32Attribute`
+    - `Int64Attribute`
+    - `NumberAttribute`
+    - `StringAttribute`
+    - `ListAttribute`
+        - `ElementType: types.BoolType`
+        - `ElementType: types.Float32Type`
+        - `ElementType: types.Float64Type`
+        - `ElementType: types.Int32Type`
+        - `ElementType: types.Int64Type`
+        - `ElementType: types.NumberType`
+        - `ElementType: types.StringType`
+
+<Note>
```

Comment:
I'd strongly encourage removing the note, [according to our style guide](https://docs.google.com/document/d/1gE5FWf_eEqoeUw4kWUht1YVucbVPXFMxnYNQ6ndlPpc/edit?tab=t.ssj87oswdn7h#bookmark=id.2zqlmzl3r7k1) we try to avoid adding callouts unless they are absolutely necessary.
```suggestion
```

(If you accept this please remove the rest of the lines below up to line 40

Reply:
Good catch, removed in 4177307aac46433d230de9eed10a6ef3488366a4 ⚡